### ISSUE TITLE
OIP-28 Decorate office with Ethereum related posters/content

### ISSUE THREAD

**Issue by franzihei** [ethdevberlin/OIPS#28]
Now that we have this amazing new office and a lot of empty frames or frames with "random" content, I would love for us to make the EF office even more Ethereum themed by using the frames we have and filling them with nice art/content. :) 

Things from the top of my head that could fit in frames:

Smaller frames:
- Front page of the Ethereum White Paper and/or Yellow Paper
- "Devcon wall" with several frames with impressions from Devcon, e.g. the different logos and locations (just like on the Devcon archive)

Bigger frames:
- Ethereum related artworks like from [Art by Nelly](https://artbynelly.com/)
- "Ethereum Infinite Garden" poster artwork by Tomo

... and maybe even more cool stuff. :)

**Comment by ligi** [comment]
Full support for this one - thanks for bringing it up. Already replaced one of these "random artworks" that I really found ugly with one in the Ethereum context and agree we should do more. Just contacted nelly if she would be OK with us hanging some of her art. Also contacted skylar - there where plans to open a devcon swag shop where you could e.g. buy devcon posters (also with tomos art) - so we could put these up.

**Comment by franzihei** [comment]
Yes I also told Tomo about the idea a few months ago already and he loved it. :)

**Comment by gumb0** [comment]
And we still have a link to Devcon 6 poster in https://github.com/ethdevberlin/OIPS/issues/21

**Comment by mvglasenapp** [comment]
I left 4 big ETHBerlin 1 posters on the big wooden table in the new meeting room and I'll get some more frames
